### ISSUE TITLE
Random tests in PottsCellTest

### ISSUE THREAD

**Issue by cainja** [bagherilab/ARCADE#70]
I occasionally get failed tests from PottsCellTest very inconsistently, both locally and using Github CI.  

Example: https://github.com/bagherilab/ARCADE/actions/runs/11398634584/job/31715987441

```
PottsCellTest > updateTarget_withRegions_updatesValues() FAILED
    org.opentest4j.AssertionFailedError at PottsCellTest.java:1039

PottsCellTest > updateTarget_defaultRegion_updatesValues() FAILED
    org.opentest4j.AssertionFailedError at PottsCellTest.java:1131
```

Am not sure what is causing the issue.

**Comment by cainja** [comment]
Another example: https://github.com/bagherilab/ARCADE/actions/runs/11412148492/job/31757662601?pr=73

**Comment by jessicasyu** [comment]
Yeah, I've been trying to figure out how to reproduce this in order to fix it. I'll keep digging...

**Comment by jessicasyu** [comment]
I figured it out -- the random volumes occasionally produced a default region volume larger than the cell volume, leading to an incorrect negative value when calculating scales.

Fixed as part of #85
